### PR TITLE
[REM] website_sale_customer_type: remove unused functionality

### PR DESCRIPTION
**Pull request by remytms**
## Description

This PR remove unused features of website_sale_customer_type. It simplify the module and will be easier to maintain.

If theses features are needed, the code should be moved on another module that depend on this one.

## Odoo task (if applicable)

[task](https://gestion.coopiteasy.be/web#id=9545&action=479&model=project.task&view_type=form&menu_id=536)

## Checklist before approval

- [ ] Tests are present (or not needed).
- [x] Credits/copyright have been changed correctly.
- [ ] Change log snippet is present.
- [ ] (If a new module) Moving this to OCA has been considered.

### CODE REVIEW

File: website_sale_customer_type/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -47,3 +37,22 @@ activated/published acquirers.
...
+To set a default Customer Type that applies on user that are not
+connected on the e-commerce, do the following:
+
+- Go to Configuration > Users
+- Remove de default filter and select the filter "Inactive Users".
+- Search for "Public" and select the "Public User".
+- Click on the partner linked to this user.
+- On the partner form set a Customer Type and save.
+
+The Customer Type set on the "Public User" will be used for user that
```
Comment (victor-champonnois) [inline website_sale_customer_type/readme/CONFIGURE.rst]: for user**s**

File: website_sale_customer_type/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -8,11 +8,6 @@ A Customer Type brings functionality to :
...
   of allowed acquirers.
-- Force the customer to login before accessing to the e-commerce.
-- Redirect the customer to a specific location after the selection of
-  its type (see Customer Type Selector Popover).
 
-All of these features can be activated or not.
-
-It also implement a Customer Type Selector that popups on the e-commerce
-page at the first visit of a new customer (if configured).
+You can chose to filter only product, only acquirers or both for each
```
Comment (victor-champonnois) [inline website_sale_customer_type/readme/DESCRIPTION.rst]: product**s**

**Review by victor-champonnois — APPROVED** [review]
LGTM, not much to review, if the functional tests are OK and no errors, it's good to go

File: website_sale_customer_type/readme/CONFIGURE.rst (hunk 1 above)
Comment (remytms) [reply website_sale_customer_type/readme/CONFIGURE.rst]: Done

File: website_sale_customer_type/readme/DESCRIPTION.rst (hunk 2 above)
Comment (remytms) [reply website_sale_customer_type/readme/DESCRIPTION.rst]: done

**Pull request merged**